LICENSE

```diff
@@ -1,4 +1,5 @@
-Copyright (C) 2022 Apple Inc. All rights reserved.
+Copyright (C) 2013-2022 Apple Inc. All rights reserved.
+Copyright (C) 2017 Google Inc. All rights reserved.
```

Comment:
LGTM (however I am not an OSS lawyer :)). Do we intend to continue updating this section as new PRs from contributing vendors come in?

Reply:
Yes, that's the idea.